### DISCUSSION TITLE
Taskbar overlapping over start menu (windows 11 style) when using right taskbar alignment

### DISCUSSION THREAD

**Discussion by ralqadri** [valinet/ExplorerPatcher#3570] [Q&A] [answered]
Self-explanatory. Is there anything I can do to fix this? 

Here's a little preview:
![image](https://github.com/user-attachments/assets/f15228b8-df8c-4567-a426-c5a85fe8c1dc)

Doesn't happen for the left-aligned taskbar:
![image](https://github.com/user-attachments/assets/450617c8-7caa-4863-abec-655011489fcd)

Both of these behaviours are consistent for each alignments. Left-aligned taskbar overlaps the start menu 99% of the time, right-aligned taskbar doesn't overlap the start menu 100% of the time. 

[There was a similar issue reported before](https://github.com/valinet/ExplorerPatcher/discussions/3528) but I do not think this is the same case as mine since OP was using the Windows 10 style start menu.

Thanks in advance!

**Comment by pyrates999** [comment, marked as answer]
I think the pre-release has it fixed: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.4_c127c05

**Reply by ralqadri** [reply]
Thank you so much! This new version fixes my problem.
